ansible/roles/hcv-haproxy-configure/templates/haproxy.cfg.j2

```diff
@@ -403,7 +403,22 @@ backend release-{{ release }}
 {%- endif %}  
 {% endfor %}
 {% for item in haproxy_facts.backends|sort(attribute='local',reverse=True) %}
-{% if item['release_number'] != release %}
+{% if item['release_number'] != release and item['exclusive_tenant'] == "" %}
```

Comment:
Isn't the new logic for when the item['exclusive_tenant'] != "" not item['exclusive_tenant'] == "" ? 
I don't completely understand it, that's why I'm not sure

Reply:
we have replaced one loop for two.  Previously, we'd loop on all shards in releases that don't match the current one.  Now we loop on all shards that don't match the current release and don't have an exclusive tenancy.  Then we go back and loop again and include any that have an exclusive tenancy but aren't part of the current release.